app/workers/rapid_ready_for_decision/disability_compensation_job.rb

```diff
@@ -52,7 +52,7 @@ def bp_readings(client)
     end
 
     def medications(client)
-      @medications ||= client.list_resource('medications')
+      @medications ||= client.list_resource('medication_requests')
```

Comment:
We use the client to query the `MedicationRequest` endpoint, but we interpret that as medications.

Reply:
Since #9207 is already merged, don't you also have to update Form526HypertensionJob as well?

Reply:
I believe I did.